src/main/kotlin/com/zepben/evolve/services/network/tracing/networktrace/conditions/OpenCondition.kt

```diff
@@ -10,12 +10,9 @@ package com.zepben.evolve.services.network.tracing.networktrace.conditions
 
 import com.zepben.evolve.cim.iec61970.base.wires.SinglePhaseKind
 import com.zepben.evolve.cim.iec61970.base.wires.Switch
-import com.zepben.evolve.services.network.tracing.networktrace.NetworkTraceQueueCondition
 import com.zepben.evolve.services.network.tracing.networktrace.NetworkTraceStep
 import com.zepben.evolve.services.network.tracing.traversal.StepContext
 
-// NOTE: This is a queue condition and not a stop condition because once cuts and jumpers are added to the network model we need the 'next' terminal
```

Comment:
I removed note because I feel it didn't reflect how we actually ended up implementing the tracing. And now thinking about it a bit more this probably could have been a stop condition, but I don't know if it's a good idea to change it because we have things relying on ifStopping having certain criteria met against your specific stop conditions added.